ui/src/views/detail/detail.tsx

```diff
@@ -69,7 +69,16 @@ export const MintedSiteDetail = () => {
                       heading="Description"
                       minH={120}
                       maxH="auto"
-                      children={<p>{description}</p>}
+                      children={
+                        <div
+                          dangerouslySetInnerHTML={{
+                            __html:
+                              '<p>' +
+                              description.replaceAll('\n', '</p><p>') +
+                              '</p>',
+                          }}
+                        />
+                      }
```

Comment:
cool thanks!

Reply:
I had to do this

```
style={{
   whiteSpace: description.includes('\n')
     ? 'pre'
     : 'normal',
   }}
```
otherwise this happened
<img width="529" alt="image" src="https://user-images.githubusercontent.com/52458215/211411590-d565d5d3-05e4-4eb4-8299-a374306425e0.png">

when it should look like this
<img width="553" alt="image" src="https://user-images.githubusercontent.com/52458215/211411652-d6577ce9-f2f5-4673-a3c5-80ce24972a50.png">